x-pack/plugins/console_extensions/spec/overrides/ilm.put_lifecycle.json

```diff
@@ -0,0 +1,54 @@
+{
+  "ilm.put_lifecycle": {
+    "data_autocomplete_rules": {
+      "phases": {
+        "warm": {
+          "after": "",
```

Comment:
if use `1d` is a value, console with give this as an example, making it clearer what syntax should be used.

Reply:
done